crates/forge/tests/it/invariant.rs

```diff
@@ -391,7 +391,7 @@ async fn test_shrink_fail_on_revert() {
     runner.test_options.fuzz.seed = Some(U256::from(119u32));
     runner.test_options.invariant.fail_on_revert = true;
     runner.test_options.invariant.runs = 1;
-    runner.test_options.invariant.depth = 100;
+    runner.test_options.invariant.depth = 1000;
```

Comment:
please undo

Reply:
I increased this because the test is failing as you can see

Reply:
cc @grandizzy

Reply:
yeah, is OK to increase if fails again, the test asserts failed sequence is reduced to 10 calls so depth doesn't matter. Going to check why it failed in CI, don't see any change that should affect it...

Reply:
it's passing now